.vscode/tasks.json

```diff
@@ -20,6 +99,81 @@
                 "${workspaceFolder}/tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj"
             ],
             "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "openapi test",
```

Comment:
i am not happy with this. Referencing a particular test just to get the openapi.json sounds like way to many moving parts.
I would recommend you use the openapitools dotnet tool instead
https://learn.microsoft.com/en-us/aspnet/core/web-api/microsoft.dotnet-openapi?view=aspnetcore-8.0

Reply:
I'll look into using the .NET openapitools. 
I found the logic for the Jellyfin openapi generation in as part of this repo Github workflow which invokes a unit test to generate the openapi.json here: https://github.com/jellyfin/jellyfin/blob/master/.github/workflows/ci-openapi.yml#L12-L35

However I do think it would be good for the project to use the standalone tool command if possible.

I also think it would be good if there was some method to auto-install the `openapitools` tool when the Jellyfin.sln file is open for the first time in vscode.

Would you have any ideas regarding method to make it easier on developers to installing the OpenAPI tool automatically?

So far the idea I am thinking of following is replacing the task to run the OpenAPI test with a task that runs the `dotnet install` command if openapitools command isn't installed.

Reply:
Actually it looks like the [dotnet-tools.json](https://github.com/jellyfin/jellyfin/blob/master/.config/dotnet-tools.json) file in the repo can be used to install that dependency.

Reply:
@JPVenson I have updated the openapi generation logic to use the [Swashbuckle CLI tool](https://github.com/domaindrivendev/Swashbuckle.AspNetCore?tab=readme-ov-file#swashbuckleaspnetcorecli) to generate the `openapi.json` file.

I have left the Github CI workflows as is and those still build and run the Server intergration test to generate the `openapi.json`.